### ISSUE TITLE
[meta] Backend for admin to handle flags

### ISSUE THREAD

**Issue by mayel** [bonfire-networks/bonfire-app#112]

**Comment by mayel** [comment]
We can already list flags. I guess the admin should be able to message the users concerned (so simply buttons to existing message functionality), delete activities/objects (depends on #79), and block actors or instances (do we have an API for that @KarenKonou?)

**Comment by mayel** [comment]
closing in favour of the more detailed issues
